server/routes/activities/create-an-activity/handlers/sessionTimes.ts

```diff
@@ -192,14 +56,11 @@ function startDateBeforeEarlierSession(customSlot: Slot, allSlots: Slot[]): bool
   )?.customStartTime
 
   // check an ED session is after any PM start time
-  if (
-    pmSlotStartTime !== undefined &&
+  return (
     customSlot.timeSlot === TimeSlot.ED &&
+    pmSlotStartTime !== undefined &&
     customSlot.customStartTime.localeCompare(pmSlotStartTime) <= 0
-  ) {
-    return true
-  }
-  return false
+  )
```

Comment:
Removed because `false` is implied if this is not `true` ?

Reply:
hard to read in the PR, this is the statement

// check an ED session is after any PM start time
return (
    customSlot.timeSlot === TimeSlot.ED &&
    pmSlotStartTime !== undefined &&
    customSlot.customStartTime.localeCompare(pmSlotStartTime) <= 0
  )